packages/core/src/components/graphs/treemap.ts

```diff
@@ -167,8 +167,7 @@ export class Treemap extends Component {
 			.attr('width', (d) => d.x1 - d.x0)
 			.attr('height', (d) => d.y1 - d.y0)
 			.style('fill', (d) => {
-				while (d.depth > 1) d = d.parent;
```

Comment:
It seems the treemap does not support nested data structures, so this `while` loop seems safe to delete.